sdk/attestation/azure-security-attestation/README.md

```diff
@@ -115,7 +115,7 @@ authentication, the documentation for that API will reflect that the attestation
 
 To interact with the authenticated APIs supported by the Azure Attestation service, your client must present an Azure Active Directory bearer token to the service.
 
-The simplest way of providing a bearer token is to use the  `ClientSecretCredential` authentication method by providing client secret credentials is being used in this
+The simplest way of providing a bearer token is to use the `EnvironmentCredential` authentication method by providing client secret credentials is being used in this
```

Comment:
This sentence reads weirdly (almost like two separate sentences are merged into one), especially "is being used in this getting started section".

Also, do we need to update `providing client secret credentials`, or is that good as-is?